Makefile

```diff
@@ -115,7 +115,7 @@ win64: check
 	@echo "\nBuilding 64bit Windows installer."
 	python win_installer.py 64 setup.py
 
-macos:
+macos: check
```

Comment:
Actually I wouldn't mind if "check" was left out the build commands, it reduces the CI time. But this could be a breaking change for developers that expect this to be part of the process.

@ntoll @tjguk @dybber @devdanzin @ZanderBrown thoughts?

Reply:
IIUC, check would still run in tests, so the effect for code that passes check would be faster builds and same amount of information. For code that doesn't pass check, it would mean the build doesn't abort earlier. So I think removing check from build is OK as the usual case is passing it. If we'd like to avoid wasting build minutes on rare check fail cases, maybe we could delay build until check passes in tests.

Reply:
Yeah, I think removing it for the CI builds is the right call, since otherwise we are running the tests twice.

My main concern if people that run builds locally would prefer to keep the tests as part of the build command, so that it continues to aborts the build if the local tests fail.

Reply:
I thought about that, too... I'm trying to keep things as much as they were before, though, and forgot to keep the `check` dependency on the `macos` target.

Future `pup` should be able to run tests **within the packaged application** but that's not the case yet... Thus, inline with @carlosperate thoughts WRT local builds, I figured I'd "play it safe" and keep the dependency.

Reply:
FYI: Logged #1248 to track / promote this discussion.